### DISCUSSION TITLE
Make it possible to quicklook files from the opening tab of another program

### DISCUSSION THREAD

**Discussion by hlvrn** [QL-Win/QuickLook#1413] [Ideas]
This is possible in Mac and is a HUGE performance helper.

Example: You're in a program (like a photo editing software), and click on open files - maybe you want to add photos to your slideshow, as an example. When you click on opening files within almost any program in Windows, it opens up the File Explorer - allowing you to search through the files you're looking for in the file explorer window, and add them to whatever program you're using. BUT, the problem is - QuickLook does not allow you to quickview the files through file explorer window.

Simply put, QuickLook doesn't allow quickviewing files from the file explorer window / tab which has been opened through another program. It only allows quickviewing files when you're using the file explorer as the main program in Windows.

This is a very significant limitation. Many times when you're interested in quickviewing a file, like a photo, or a video, you're inside another program, and you're browsing through many files which you may want to add to the program you're using. Now, you have to go out of your program, open a normal, main file explorer window, to view what files you want to add you program. Then you have to go back to your program, remember the files you just quickviewed from the main file explorer window, and retrace them through the program. This is extremely cumbersome.

I hope this post is helpful and can lead to a new feature which is sorely missing.

**Comment by rabelux** [comment]
Already possible. See here:
https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions

**Reply by hlvrn** [reply]
I've installed msi version and unfortunately that feature does not work.

If you're within a window, it's not possible to preview a file.

A bug?
